service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/PractitionerMapper.java

```diff
@@ -0,0 +1,72 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import static org.hl7.fhir.dstu3.model.IdType.newRandomUuid;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import uk.nhs.connect.iucds.cda.ucr.AD;
+import uk.nhs.connect.iucds.cda.ucr.PN;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01AssignedEntity;
+import uk.nhs.connect.iucds.cda.ucr.TEL;
+
+import org.hl7.fhir.dstu3.model.Address;
+import org.hl7.fhir.dstu3.model.ContactPoint;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PractitionerMapper {
+
+    @Autowired
+    HumanNameMapper humanNameMapper;
+
+    @Autowired
+    ContactPointMapper contactPointMapper;
```

Comment:
See Java coding standards. Let's try to use constructors instead of field injection.
Field should be private